data/en/bots/268420199370194944.md

```diff
@@ -0,0 +1,19 @@
+---
+avatar: >-
+  https://images-ext-2.discordapp.net/external/Hp0JL_e3p4hBO5LwX6pdBfm6N3fmg1Ko3OsIHP0qkMI/https/cdn.discordapp.com/avatars/268420199370194944/3fe0f006f027991e3cdb64eabbeb542d.png
+cover: null
+custom_path: bluebot
+description: >-
+  Customizale soundboard & general purpose commands - spam your friends with
+  audio memes
+github:
+  owner: thibautbessone
+  repo: DiscordBlueBot
+images: []
+link: >-
+  https://discordapp.com/oauth2/authorize?client_id=268420199370194944&scope=bot&permissions=-1
+nsfw: false
+pagename: BlueBot
+prefix: '! (Customizable)'
+support: 'https://discord.gg/rSekkJv'
+---
```

Comment:
Not even a long description?

Reply:
I assumed you could edit the description later, I can't really do it right now - if you actually can't edit the description later, reject this PR and I'll make another one when I have time

Reply:
You can, but that would require another PR for updating it.
So it's completely your choice.

Reply:
Alright - please accept this one then, I don't really know when I'll take the time to write a more exhaustive description. Thanks !